src/Build/BackEnd/Components/SdkResolution/NuGet/GlobalJsonReader.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Shared;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+using SdkResolverContextBase = Microsoft.Build.Framework.SdkResolverContext;
+
+namespace Microsoft.Build.BackEnd.SdkResolution.NuGet
+{
+    /// <summary>
+    /// Reads MSBuild related sections from a global.json.
+    /// </summary>
+    internal static class GlobalJsonReader
+    {
+        public const string GlobalJsonFileName = "global.json";
+
+        public const string MSBuildSdksPropertyName = "msbuild-sdks";
```

Comment:
Is this new? Is it [documented](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json)?

Reply:
It is so new, that it isn't documented yet.

Reply:
That's cool, but it needs to be. Can we track that work somewhere?

Reply:
I've added it as an item in my spec: https://github.com/Microsoft/msbuild/issues/2803